apis/core/lib/domain/dataset/update.ts

```diff
@@ -1,19 +1,22 @@
 import { NamedNode } from 'rdf-js'
-import { cc } from '@cube-creator/core/namespace'
+import { cc, lindasSchema } from '@cube-creator/core/namespace'
 import { dcat, hydra, rdf, schema, vcard, _void } from '@tpluscode/rdf-ns-builders'
 import { GraphPointer } from 'clownface'
 import { ResourceStore } from '../../ResourceStore'
+import * as projectQuery from '../cube-projects/queries'
 
 interface AddMetaDataCommand {
   dataset: GraphPointer<NamedNode>
   resource: GraphPointer
   store: ResourceStore
+  findProject?(arg: { dataset: GraphPointer }): Promise<NamedNode | undefined>
 }
 
 export async function update({
```

Comment:
This is minor, but sometimes in the code I was wondering, 'update what?', what about 'updateProject' ?

Reply:
I see. The "full picture" is like

```js
import { update } from './dataset.js'
```

But yea, as soon as you import multiple such methods one needs to become creative.